content/EDITING.md

```diff
@@ -228,6 +229,30 @@ Quotes: items with double quotes in them should be wrapped in single quotes, oth
 
 Items containing special characters (colons, brackets, etc) may need to be wrapped in double quotes.
 
+<a id="foreign-languages"></a>
+## Foreign Languages
+
+Text in foreign language should be entered in HTML and include the `lang` and `hreflang` attributes. You can search for the appropriate language tag via [the Language Subtag Lookup](https://r12a.github.io/app-subtags/) page. Example:
```

Comment:
That's a good start! Maybe JAWS support it and automatically will use another language. But it didn't work with Orca in the updated preview (thanks a lot for the quick delivery of changes through netlify, great idea!).

@lunacodes an alternative might be using [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)? I quickly edited the page locally to use an aria label, and refreshed with Orca running. Then selected the text in that bullet item so Orca would read it for me. It read "Bullet. Introduction to CWL in Japanese is a 15 minute introduction to the CWL project in Japanese." (it normally says "blabla. link." but it omitted the link part, might be a bug in Orca when an `aria-label` is used in a link, I will follow-up in their gnome issue tracking later).

I think it should be OK to use an `aria-label` in English, since the page I am reading also contains a `<html lang="en">` declaration at the top (i.e. the text of the page is in English, so it wouldn't be a surprise to a screen reader user to have the translations for short text like this).

WDYT?

Reply:
Looks like Orca sometimes reads the text content ignoring HTML content links links. But if I press `L` to go into the list mode, and navigate the link with arrow keys, then it reads the text correctly, reading the `aria-label`, followed by the word `link` and the rest of the item.

Reply:
@kinow I tried it out w/ NVDA (windows screen reader) as well, and had pretty much the same experience as you. The `aria-label` seems like the way to go. The `lang` and `hreflang` still need to be specified as per the WCAG guidelines (https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang#accessibility), but they can be used together just fine, afaict. 

I'll add it in as "Intro to CWL in Japanese" for now

Reply:
I have a Windows box, but never used NVDA! Going to try that next time I use it (which is really sporadically, last time I used it was probably in October or November :laughing: ). From what I heard NVDA is a better product than Orca (feature-wise), but I like Orca for being open and being around for a long time in gnome.

+1 on the translated text. Thanks @lunacodes !